docker-compose.yml

```diff
@@ -77,6 +77,7 @@ services:
       depends_on:
         - zookeeper
     rediscluster:
+        platform: linux/amd64
```

Comment:
how does specifying this platform allow running this on arm laptops?

Reply:
Docker on M1s can run amd64 through emulation. idk if you remember we used to not publish an arm image for `testrunner` and it was _so slow_.

We also define this platform setting for other images already in this docker-compose.yml (mysql)